modules/network/vpc/variables.tf

```diff
@@ -250,3 +266,12 @@ variable "firewall_log_config" {
     error_message = "var.firewall_log_config must be set to \"DISABLE_LOGGING\", or enable logging with \"INCLUDE_ALL_METADATA\" or \"EXCLUDE_ALL_METADATA\""
   }
 }
+
+resource "terraform_data" "secondary_ranges_validation" {
+  lifecycle {
+    precondition {
+      condition     = length(var.secondary_ranges) == 0 || length(var.secondary_ranges_list) == 0
+      error_message = "Only one of var.secondary_ranges or var.secondary_ranges_list should be specified"
```

Comment:
Can we encourage customers to migrate and remove var.secondary_range from the error message? If needed, maybe link to an example on how the secondary_ranges_list needs to be defined.

Reply:
This message is only for users who erroneously try to use both the list and the map. This can't be a warning, has to be an error